cluster/src/main/java/com/hartwig/pipeline/execution/vm/VirtualMachineJobDefinition.java

```diff
@@ -2,13 +2,14 @@
 
 import com.hartwig.pipeline.ResultsDirectory;
 import com.hartwig.pipeline.execution.JobDefinition;
+import com.hartwig.pipeline.tools.Versions;
 
 import org.immutables.value.Value;
 
 @Value.Immutable
 public interface VirtualMachineJobDefinition extends JobDefinition<VirtualMachinePerformanceProfile> {
 
-    String STANDARD_IMAGE = "diskimager-standard";
+    String STANDARD_IMAGE = "pipeline5-" +Versions.imageVersion();
```

Comment:
Could we just move all this into `Versions`?

Reply:
Good idea